packages/web-react/src/components/Tooltip/__tests__/useFloatingUI.test.ts

```diff
@@ -0,0 +1,30 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useFloatingUI } from '../useFloatingUI';
+
+describe('useFloatingUI', () => {
+  it('should initialize with default values', () => {
+    const { result } = renderHook(() =>
+      useFloatingUI({
+        arrowRef: { current: null },
+        flipCrossAxis: false,
+        flipFallbackAxisSideDirection: 'none',
+        flipFallbackPlacements: 'bottom',
+        flipProp: false,
+        isOpen: false,
+        onToggle: jest.fn(),
+        shiftProp: false,
+        sizeProp: false,
+        tooltipPlacement: undefined,
+      }),
+    );
+
+    expect(result.current.context).toBeDefined();
```

Comment:
Does it return some data or it is only defined? Maybe using default values instead of just defined will be a better example of what data it returns.

Reply:
added some default values, but there is a lot of data and functions generated by the library and it is hard to test